app/main.py

```diff
@@ -1,2 +1,117 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.dict_as_list = [(), (), (), (), (), (), (), ()]
+        self.cells = len(self.dict_as_list)
+        self.max_elements = 5
+
+    def __setitem__(self,
+                    key: Any,
```

Comment:
```suggestion
                    key: Hashable,
```
please fix everywhere